plugins/webapi/mana/pastconsensusbasemanavector.go

```diff
@@ -0,0 +1,44 @@
+package mana
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/iotaledger/goshimmer/packages/mana"
+	manaPlugin "github.com/iotaledger/goshimmer/plugins/mana"
+	"github.com/labstack/echo"
+)
+
+// getPastConsensusManaVectorHandler handles the request.
+func getPastConsensusManaVectorHandler(c echo.Context) error {
+	var req PastConsensusManaVectorRequest
+	if err := c.Bind(&req); err != nil {
+		return c.JSON(http.StatusBadRequest, PendingResponse{Error: err.Error()})
+	}
+	timestamp := time.Unix(req.Timestamp, 0)
+	consensus, _, err := manaPlugin.GetPastConsensusManaVector(timestamp)
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, PastConsensusManaVectorResponse{Error: err.Error()})
+	}
+	manaMap, err := consensus.GetManaMap()
```

Comment:
`GetManaMap()` calls `getMana()` under the hood, that was designed for real time tracking so it updates the `BaseMana` for the node wrt to current time. This is obviously not what we want. Add an `update` flag to `getMana()` arguments to be able to disable updating when needed.